pilot/pkg/networking/core/v1alpha3/sidecar_simulation_test.go

```diff
@@ -1626,8 +1626,8 @@ metadata:
   creationTimestamp: "{{.Time}}"
 spec:
   parentRefs:
-  - kind: Mesh
-    name: istio
+  - kind: Gateway
```

Comment:
this should be Mesh route, not gateway. But its hard to test now since we don't support arbitrary hostnames in the GEP yet, which the test is expecting...

Reply:
I disabled this test for now. We can add it back once we have support for arbitrary hostnames in mesh routes.